docs/docsite/rst/dev_guide/testing/sanity/import.rst

```diff
@@ -10,6 +10,17 @@ This enables support for sanity tests such as :ref:`testing_validate-modules` an
 Handling import errors
 ----------------------
 
+Ansible executes across many hosts and can uses different interpreters (normally python) that can even have multiple versions installed in parallel at the same time.
```

Comment:
Since the import tests only apply to Python, I don't think we need to mention non-Python interpreters here.

```suggestion
Ansible executes across many hosts and can use multiple Python interpreters at the same time, which may even have different versions.
```